core/src/main/java/feast/core/service/SpecService.java

```diff
@@ -625,4 +634,110 @@ public UpdateStoreResponse updateStore(UpdateStoreRequest updateStoreRequest)
         .setStore(updateStoreRequest.getStore())
         .build();
   }
+
+  /**
+   * Applies the given Feature Table to the Feature Table registry. Creates the Feature Table if
+   * does not exist, otherwise updates the existing feature Table. Applies feature table in project
+   * if specified, otherwise in default project.
+   *
+   * @param request contain apply feature Table parameters.
```

Comment:
`contain apply feature Table parameters.` is quite poorly worded

Reply:
Re-worded the description.